show-me-the-visualization/app.html

```diff
@@ -70,6 +87,7 @@
         "$schema": "https://vega.github.io/schema/vega-lite/v4.json",
         "width": 800,
         "height": 600,
+        "title": {"text": "Iterative rootfinding", "fontSize": 20, "fontWeight": "normal"},
```

Comment:
This title is not shown in the README. Please add there as well

Reply:
done